ISISComputingGroup/ibex_gui
# Ticket2074: Synoptics not appearing

### Description of work

This ticket contains a large amount of refactoring for how synoptics are compared to schema in the GUI. * The GUI no longer checks synoptics are correct by the schema when loading them to display as the BlockServer would have already done this check.
* The GUI only checks a synoptic against the schema once when it saves and will give a better error message to the user if this check fails.

This will probably fix https://github.com/ISISComputingGroup/IBEX/issues/2074 and https://github.com/ISISComputingGroup/IBEX/issues/2090 as well as the exception referenced in the comments of 2090. However, these bugs are hard to reproduce so good luck!

### To test

Change the schema on the blockserver, confirm that the GUI behaves more nicely (doesn't throw errors), confirm that when saving a new schema a sensible error is displayed to the user. Try and reproduce the tickets referenced above.

---

#### Code Review


### Functional Tests


### Final Steps
